app/app.go

```diff
@@ -103,6 +104,8 @@ func Run(ctx context.Context, kubeConfig rest.Config, cfg *Config) error {
 	// cache works properly
 	features.InitializeFeatures(scaledContext, cfg.Features)
 
+	metrics.Register(scaledContext)
```

Comment:
> It looks like monitoring with always run now even if the env vars are not set
right, rancher metrics is out of control in current PR. it will always enable.

Only we can control is whether if we will enable following each norman metric or not.
 - NORMAN_SESSION_SERVER_METRICS
 - NORMAN_GENERIC_CONTROLLER_METRICS

Do you want to add another environment variable to control if rancher collect rancher internal metrics or not at all? 
Or do you want me to delete just PrometheusMetrics variable ? since it's always true.

Reply:
I think this should be a setting to make this a top-level option in the GUI. https://github.com/rancher/rancher/blob/a0b4e1b1f363d906a8daebabf65f0e9a5fc96482/pkg/settings/setting.go

Reply:
Sure let me fix

Reply:
keep it an env var. Gernaly the expectation with a setting is that you can change it at run time. You wont be able to do that with this.
Kind of a separate but related question: is there a real practical and real-life reason for having two different metric env vars? Could we condense down to a single one? If so, I would go with `CATTLE_PROMETHEUS_METRICS`.